src/data-access/repositories/mongo/mongo-ad-lib-actions-repository.ts

```diff
@@ -20,15 +21,16 @@ export class MongoAdLibActionsRepository extends BaseMongoRepository implements
     return AD_LIB_ACTIONS_COLLECTION
   }
 
-  public async getActionManifests(): Promise<ActionManifest[]> {
+  public async getActionManifests(rundownId: string): Promise<ActionManifest[]> {
     this.assertDatabaseConnection(this.getActionManifests.name)
-    const adLibActions: AdLibAction[] = await this.getCollection().find<AdLibAction>({}).toArray()
-    return adLibActions.map(adLibAction => this.mapToActionManifest(adLibAction))
+    const adLibActions: AdLibAction[] = await this.getCollection().find<AdLibAction>({rundownId: rundownId}).toArray()
+    return adLibActions.filter(action => action.rundownId === rundownId).map(adLibAction => this.mapToActionManifest(adLibAction))
```

Comment:
Why is the adLibActions filtered by rundownId? That seems unnecessary.

Reply:
Woops. Forgot to remove that when i changed the filterint to happen at database level. Gone now.